libraries/api/src/middleware/handle.js

```diff
@@ -12,7 +12,7 @@ const callHandler = ({ entry, context, monitor }) => {
     Promise.resolve(null).then(() => {
       return entry.handler.call(req.tcContext, req, res);
     }).then(() => {
-      if (!req.public && !req.hasAuthed) {
+      if (!req.public && !req.satisfyingScopes) {
```

Comment:
Now that I look at this, won't this throw an error in sentry every time a request is made that doesn't have enough scopes? Checking `authenticated` here was the actual intent iirc.